### PR TITLE
2.5 readme

### PR DESCRIPTION
**Pull request by craigburke**

### CODE REVIEW

File: asset-pipeline-gradle/src/main/groovy/asset/pipeline/gradle/AssetCompile.groovy [hunk 1]
```diff
@@ -165,27 +165,45 @@ class AssetCompile extends DefaultTask {
     @TaskAction
     @CompileDynamic
     void compile() {
-        // println "Compiling assets in directory ${assetsDir}"
-        def resolver = new FileSystemAssetResolver('application', assetsDir.canonicalPath)
-        AssetPipelineConfigHolder.registerResolver(resolver)
-        
-        //Time to register Jar Resolvers
-        this.getClasspath()?.files?.each { file ->
```
Comment (davydotcom) [inline asset-pipeline-gradle/src/main/groovy/asset/pipeline/gradle/AssetCompile.groovy]: not sure why this part was removed?

File: asset-pipeline-gradle/src/main/groovy/asset/pipeline/gradle/AssetCompile.groovy [hunk 2]
```diff
@@ -165,27 +165,45 @@ class AssetCompile extends DefaultTask {
...
+            if (isJarFile) {
+                registerJarResolvers(resolverFile)
+            }
+            else if (isAssetFolder) {
+                def fileResolver = new FileSystemAssetResolver(path, path, false)
+                AssetPipelineConfigHolder.registerResolver(fileResolver)
+            }
+        }
+
+        this.getClasspath()?.files?.each { registerJarResolvers(it) }
```
Comment (davydotcom) [inline asset-pipeline-gradle/src/main/groovy/asset/pipeline/gradle/AssetCompile.groovy]: aha thats why

Comment (craigburke) [inline asset-pipeline-gradle/src/main/groovy/asset/pipeline/gradle/AssetCompile.groovy]: yes sir, that is correct.

Comment (craigburke) [inline asset-pipeline-gradle/src/main/groovy/asset/pipeline/gradle/AssetCompile.groovy]: Also, don't ever question me again.

**Pull request merged**